src/sentry/static/sentry/app/components/assistant/helper.jsx

```diff
@@ -46,6 +47,16 @@ const AssistantHelper = createReactClass({
   },
 
   handleDrawerOpen() {
+    let {currentGuide} = this.state;
+
+    HookStore.get('analytics:event').forEach(cb =>
+      cb('assistant.guide', {
```

Comment:
We should discuss what this should be (whether action should be a part of this, so  `event_name` might be `assistant.opened`)